src/runtime_src/core/edge/user/aie/graph.cpp

```diff
@@ -48,10 +48,13 @@ zynqaie::Aie* getAieArray()
 
 namespace zynqaie {
 
-graph_type::
-graph_type(std::shared_ptr<xrt_core::device> dev, const uuid_t uuid, const std::string& graph_name, xrt::graph::access_mode am)
-  : device(std::move(dev)), name(graph_name)
+graph_instance::
+graph_instance(std::shared_ptr<xrt_core::device> dev, const std::string& graph_name,
+              xrt::graph::access_mode am, const zynqaie::hwctx_object* hwctx, const xrt_core::uuid uuid)
+  : device{std::move(dev)}, name{graph_name}, m_hwctxHandle{hwctx}
```

Comment:
Since I believe you are copying the string (`graph_name`) clang-tidy will complain that it should have been passed by value and moved.